docs/src/main/asciidoc/security-oidc-code-flow-authentication-concept.adoc

```diff
@@ -45,12 +46,282 @@ For information about how to support multiple tenants, see xref:security-openid-
 
 == Using the authorization code flow mechanism
 
+=== Configuring access to the OIDC Provider endpoint
+
+OIDC `web-app` application needs to know the endpoint URLs of OpenID Connect provider's authorization, token, `JsonWebKey` (JWK) set and possibly `UserInfo`, introspection and end session (RP-initiated logout).
```

Comment:
Use abbreviation, to reduce wordiness unless it's the first time on the page or large section.

```suggestion
The OIDC `web-app` application requires the endpoint URLs of the OIDC provider's authorization, token, `JsonWebKey` (JWK) set, and possibly the `UserInfo`, introspection and end session (RP-initiated logout).
```

Reply:
@michelle-purcell This sentence does not read right (your proposal is fine, but `endpoint` is in the wrong place), so I propose
```
The OIDC `web-app` application requires URLs of the OIDC provider's authorization, token, `JsonWebKey` (JWK) set, and possibly the `UserInfo`, introspection and end session (RP-initiated logout) endpoints.
```

Reply:
Thanks for explaining this. I created a new suggestion with your rewrite.